### DISCUSSION TITLE
Type 'bool' with x-ref is not correctly rendered

### DISCUSSION THREAD

**Discussion by mscheltienne** [pradyunsg/furo#506] [Potential Issue] [answered]
Hello,

I've been playing with your theme for a while on a sandbox repository: https://github.com/mscheltienne/simple-stimuli
I've noticed this weird [render](https://mscheltienne.github.io/simple-stimuli/dev/api/generated/stimuli.set_log_level.html#stimuli.set_log_level) today:

![image](https://user-images.githubusercontent.com/73893616/184637176-83fde41c-6040-4e47-8036-24898f4987fd.png)

Any idea why `bool` got rendered differently?

Thanks,
Mathieu

**Comment by pradyunsg** [comment, marked as answer]
> Any idea why bool got rendered differently?

That's likely something to do with how Sphinx resolved to bool reference. What exactly? No idea.

Using `python -m sphinx.ext.intersphinx https://docs.python.org/3/objects.inv`, I can see that this resolved within `std:label`:

```
        bltin-boolean-values                     Boolean Values                          : library/stdtypes.html#bltin-boolean-values
```

Why? No idea. There's a `bool` that links to the right page as well, with the right type of reference:

```
        bool                                     library/functions.html#bool
```

Why did it not link to that? no idea. Your best bet is to reach out to the authors of Sphinx (or the Sphinx extension you're using using to generate the API documentation) to look into what's happening here.

---

At the end of the day, that's more of a thing to do with how Sphinx reference resolution worked in this case, rather than how the theme has rendered the markup that Sphinx generated for it.

**Reply by mscheltienne** [reply]
Alright, thank you for looking into it. I thought it was a render issue, clearly not!
